### ISSUE TITLE
Unable to resize outside window

### ISSUE THREAD

**Issue by owcz** [yarl/pattypan#28]
![screenshot](http://i.imgur.com/lf7VMAy.png)

Unable to resize the outer edge—not sure if it's specific to the program or to the Java implementation. In any event, might be worthwhile to make the window bigger since the contents will always need to be expanded to fit properly.

Downloaded .jar release, run with `java -jar pattypan.jar`

**Comment by yarl** [comment]
Thats strange, looks like #justjavathings. So you cannot resize window like on screenshot below?

![img](https://i.imgur.com/vwU7jJZ.png)

It's confirmed working on Win, Mac and Ubuntu. I will need your distribution name and JRE/JDK. I know that Oracle one is working fine.

**Comment by owcz** [comment]
Nope, can't resize like that—I'll get the icon to resize at the edges but it doesn't do anything when I click & drag

>$ java -version
openjdk version "1.8.0_121"
OpenJDK Runtime Environment (build 1.8.0_121-b13)
OpenJDK 64-Bit Server VM (build 25.121-b13, mixed mode)

>OS: Manjaro 17.0 Gellivara (Arch)
Kernel: x86_64 Linux 4.8.17-1-MANJARO

**Comment by yarl** [comment]
For now, can you try to resize window using `config.properties` file? You should find it in `~/.pattypan` directory

**Comment by owcz** [comment]
Yes, that worked and oddly enough, I can resize the window up choosing the directory (even before clicking "next"). After the directory is chosen and its content composition displayed, the resize icon will move the whole window rather than allowing it to be resized.

**Comment by yarl** [comment]
@owcz I understand it's still the case, right? Can you check log file in the latest version of Pattypan? Maybe there is information about some exceptions.

**Comment by owcz** [comment]
Likely won't get to this for a bit, so feel free to close if needed (Xfce has all kinds of incongruities with file select so who knows)

Will test next time I batch upload

**Comment by yarl** [comment]
Switched to low prio in the meantime.
